perun-db/postgres.sql

```diff
@@ -132,7 +132,7 @@ create table cabinet_thanks (
 -- FACILITIES - sources, devices - includes clusters,hosts,storages...
 create table facilities (
 	id integer not null,
-	name varchar(128) not null, --unique name of facility
+	name varchar(128) not null, --unique name of service
```

Comment:
I do not see the reason to change the comment for column name in table facilities to "service". It looks like some unintended mass replacement.